src/components/Inputs/ExchangeAmountInput/style.ts

```diff
@@ -28,7 +28,8 @@ const useStyles = makeStyles(() => ({
     }
   },
   select: {
-    marginRight: 20
+    marginRight: '8px',
```

Comment:
Have you checked how this would affect any other places where this component is used?